src/iuse.cpp

```diff
@@ -3755,6 +3755,14 @@ int iuse::horn_bicycle(player *p, item *it, bool, const tripoint &pos)
     return it->type->charges_to_use();
 }
 
+
+int iuse::tweet(player *p, item *it, bool, const tripoint &pos)
+{
+    sounds::sound(pos, 38, _("FWEEEET!"));
+    p->add_msg_if_player(_("You blow the whistle."));
```

Comment:
Generic messages are preferable: `p->add_msg_if_player(_("You blow the %s."), it->tname().c_str());`